src/patterns/components/slideshow/base.hbs

```diff
@@ -0,0 +1,45 @@
+<div class="u-bgGray u-posRelative Slideshow js-Slideshow u-pullSides1">
+  <h2 class="u-hiddenVisually">Fun facts about Aileen</h2>
+  <ul class="u-flex u-listUnstyled Slideshow-slides js-Slideshow-slides u-padEnds2"  id="Slideshow-slides">
+    {{#each drizzle.data.fun-facts.contents.aileen}}
+      <li class="Grid Grid--withGutter Grid--alignMiddle Slideshow-slide js-Slideshow-slide u-flexExpand u-flexJustifyCenter u-sm-flexJustifyStart u-padSides5 {{#if @first}}is-visible{{/if}}">
```

Comment:
Because this content is already structured with headings, the list seems possibly unnecessary? It's not _wrong_, but I'm also not sure it lends this area much semantic value?